src/fitnesse/slim/converters/ConverterRegistry.java

```diff
@@ -50,7 +50,8 @@ protected static void addStandardConverters() {
   }
 
   public static <T> Converter<T> getConverterForClass(Class<? extends T> clazz) {
```

Comment:
I found it can also be fixed by changing `Class<? extends T>` to `Class<T>`. That might be nicer than introducing another cast.